Makefile

```diff
@@ -0,0 +1,10 @@
+# This Makefile is used to compile an LND binary for the integration tests.
+
+GO_BUILD := go build
```

Comment:
This will fail if golang isn't installed?

Reply:
True, should we just add instructions explaining that Go is a dependency?